File: Frontend/src/app/shared/api/models/sign-up-dto.ts
/* tslint:disable */
/* eslint-disable */
import { SignUpStatus } from './sign-up-status';
export interface SignUpDto {
  alsoKnownAs?: null | string;
  createdAt?: string;
  id?: number;
  status?: SignUpStatus;
  username?: null | string;
}

File: Webapi/Webapi/Models/Events/SignUp.dto.cs
namespace Webapi.Models.Events;

#pragma warning disable CS8618 // data transfer object
public class SignUpDto
{
  public int Id { get; set; }

  public DateTime CreatedAt { get; set; }

  public string Username { get; set; }

  public string? AlsoKnownAs { get; set; }

  public SignUpStatus Status { get; set; }
}